sphinx/source/docs/user_guide/tools.rst

```diff
@@ -100,10 +100,10 @@ containing tool shortcut names:
 
     tools = "pan,wheel_zoom,box_zoom,reset"
 
-However, this method does not allow setting properties of the tools.
+This method does not allow setting properties of the tools.
 
-Finally, it is also always possible to add new tools to a plot by passing
-a tool object to the ``add_tools`` method of a plot. This can also be done
+Finally, it is possible to add new tools to a plot by passing
+a tool object to the ``add_tools`` method of a plot. This can be done
 in conjunction with the ``tools`` keyword described above:
```

Comment:
Avoiding passive voice can sometimes also lead to shorter sentences. For example:

>You also have the option to add more tools to an existing plot by using the ``add_tools()`` method of your plot. You can combine the ``add_tools()`` method with the ``figure()`` function's ``tools`` argument:

Also, I'd recommend choosing more common English words (such as "combine") over less frequently used ones ("conjunction") - keeping in mind that many users of Bokeh and of Bokeh's docs don't use English as their first language.